src/sentry/templates/sentry/emails/reports/new.html

```diff
@@ -453,6 +455,16 @@ <h1 class="total-count">{{ series.legend.total.data.accepted_transactions|small_
         </tr>
       </table>
       </td>
+    {% else %}
+      <td class="project-breakdown-graph-cell transactions-empty">
+        <div style="border: 1px solid #c4c4cc; border-radius: 4px; padding: 24px 16px; text-align: center; height: 170px;">
+          <img src="{% absolute_asset_url 'sentry' 'images/email/icon-circle-lightning.png' %}" width="32px" height="32px" alt="Sentry">
+          <h1 style="font-weight: bold; font-size: 17px;">Missing transactions?</h1>
+          <p style="font-size: 11px;">With a simple snipped of code, you can see which parts of your app have the most improvements. the worst repgressions. Set it up now.</p>
+          <a href="https://docs.sentry.io/product/performance/getting-started/" class="btn" style="margin-top: 8px;">Setup Performance</a>
```

Comment:
Should this link to the docs or the app?

Reply:
@Neo-Zhixing the app would be better